internal/compiler/tests/syntax_tests.rs

```diff
@@ -293,5 +293,14 @@ Foo := Rectangle foo { x:0px; }
         "\r\nFoo := Rectangle foo { x:0px; }\r\n//               ^error{expected '\\{'}\r\n"
     )?);
 
+    // A deprecated message should be displayed
+    assert!(!process(
+        r#"
+
+Foo := FocusScope { key-pressed(event) => { if (event.text == Keys.Escape) { debug("Esc key was pressed") } accept } }
+//              Awarning{The property 'Keys' has been deprecated. Please use 'Key' instead}
+    "#
+    )?);
```

Comment:
I meant a file somewhere in the syntax_tests repository.

(this is the test that actually test the test :-) .   and this particular test don't work because it test that the test fails because it says `Awarning` instead of `warning`